blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,281 @@
+as a general-purpose computing platform. It does that through a programming and operational model
+that spans both infrastructure and application across all cloud providers and services.
+
+Wing Cloud is a unified programming and operational model that spans all providers and services. It
+celebrates the choice and flexibility the cloud has to offer without compromising the developer
+experience and engineering practices.
+
+So how does that look like in practice? Our vision for Wing Cloud consists of three main components:
+
+1. A programming language for the cloud, aptly called **Winglang**.
+2. An high level cloud SDK, aptly called **Wing Cloud Library**.
+3. An application-centric operational console, aptly called **Wing Console**.
+
+### Why are we building a programming language?
+
+With **Winglang**, developers are able to write code that includes both infrastructure **and**
+runtime code. This means that when a cloud resource is needed, developers can just add it to their
+ code and interact with it in the same way they interact with in-memory objects in traditional
+languages. The Winglang compiler takes care of the mechanics required to provision the resource,
+configure the minimal IAM permissions and wire up the infrastructure configuration so your code can
+interact with this resource at runtime.
+
+You might be wondering why this couldn't be implemented as a library within an existing language.
+This is because existing languages don't have language constructs that can express the distributed
+nature of cloud applications. To address this, Winglang has two execution phases: *preflight* and
+*inflight*. The preflight phase defines your application's infrastructure and the inflight phase
+defines its runtime behavior.
+
+Another way to think about it is *space* versus *time*. Think about programming languages as
+storytelling devices. Traditional languages are designed to tell a linear story: first this
+happened, then that, and so on. Basically the traditional purpose of programming languages was to
+describe a sequence of instructions for the CPU. They tells the story of a program over **time**.
+
+But when it comes to cloud applications, the story is not linear. There are multiple storylines
+running together, interacting with and affecting each other. It's not just about what happens
+*when*, but also *where*. There's a *spatial dimension* to account for, and it's a dimension our
+current languages are just don't designed to express.
+
+But preflights and inflights are just the tip of the iceberg in terms of what a language for the
+cloud can offer. Think about things like API endpoints and clients, telemetry, metrics and alarms,
+data schemas, and more. We have plans to bake many of these capabilities into the language and the
+standard library so that developers can truly focus on building their applications.
+
+### A standard library for the cloud
```

Comment:
I think this could be a great place to mention "wasi-cloud-core" / WebAssembly System Interface as a related technology, to reinforce the idea that we're not the only people that believe a common cloud API layer is something that is feasible / valuable: https://github.com/WebAssembly/wasi-cloud-core

Reply:
OK, added...